examples/react/contexts/WalletSelectorContext.tsx

```diff
@@ -43,6 +44,7 @@ export const WalletSelectorContextProvider: React.FC = ({ children }) => {
       modules: [
         ...(await setupDefaultWallets()),
         setupNearWallet(),
+        setupCoin98Wallet(),
```

Comment:
Can you also add Coin98 to the angular example as well.

Reply:
commented fixed